app/components/UserList/UserList.tsx

```diff
@@ -1,12 +1,52 @@
 import _ from 'lodash';
-import React, { useMemo } from 'react';
-import { useSelector } from 'react-redux';
+import React, { createRef, useMemo } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { appConfigSelector } from '../../modules/appConfig';
 import UserListItem from './UserListItem';
+import { conferenceSelector, setMiniRoom } from '../../modules/conference';
+import classes from './UserList.css';
+
+// eslint-disable-next-line react/prop-types
+const ConferenceSwitchButton = ({ miniRoom, currentMiniRoom }) => {
+  const dispatch = useDispatch();
+
+  const buttonRef = createRef<HTMLButtonElement>();
+
+  function handleClick() {
+    dispatch(setMiniRoom(miniRoom));
+    if (buttonRef.current) buttonRef.current.blur();
+  }
+
+  let buttonClasses = 'btn-dark';
+  if (miniRoom === currentMiniRoom) {
+    buttonClasses = 'btn-success active';
+  }
+  return (
+    <button
+      ref={buttonRef}
+      type="button"
+      className={`btn btn-sm ${buttonClasses}`}
+      onClick={handleClick}
+    >
+      {miniRoom}
+    </button>
+  );
+};
 
 const UserList = () => {
   const { playedBy, activeUsers, machineId } = useSelector(appConfigSelector);
 
+  const { jitsiConferenceJoined } = useSelector(conferenceSelector);
+
+  const localConferenceUser = useSelector((state) => {
+    const { jitsiParticipants } = conferenceSelector(state);
+    return _.find(jitsiParticipants, (p) => p.machineId === machineId);
+  });
```

Comment:
Maybe it would be good to create a selector with reselect for these case

```javascript
export const localConferenceUserSelector = (userMachineId) => createSelector(
    conferenceSelector,
    ({ jitsiParticipants }) => {
       return _.find(jitsiParticipants, ({ machined }) => machineId === userMachineId);
    }
)
```

and than it can be user in here like this 

```suggestion
  const localConferenceUser = useSelector(localConferenceUserSelector(machineId));
```